scalafmt-tests/src/test/resources/align/AlignByRightName.stat

```diff
@@ -73,11 +73,11 @@ libraryDependencies ++= Seq(
 libraryDependencies ++= Seq(
 "io.get-coursier" % "interface" %% "0.0.17",
 "com.geirsson1111111" %%% "metaconfig-core" % foo(a %% b) % Test,
-"org.scalacheck" %% "scalacheck" % sclacheckV % Test
+"org.scalacheck" %% "scalacheck" % scalacheckV % Test
 )
 >>>
 libraryDependencies ++= Seq(
   "io.get-coursier"       % "interface"      %% "0.0.17",
   "com.geirsson1111111" %%% "metaconfig-core" % foo(a %% b) % Test,
-  "org.scalacheck"       %% "scalacheck"      % sclacheckV  % Test
+  "org.scalacheck"       %% "scalacheck"      % scalacheckV  % Test
```

Comment:
afaict, this token isn't defined anywhere and would have come from another file. But there doesn't appear to be any good reason to misspell `scala`.